starknet_py/compile/compiler_test.py

```diff
@@ -31,7 +37,7 @@ def test_compile_direct_load():
 
 def test_compile_file_load():
     output_file_str = Compiler(
-        contract_source=[test_file_path.resolve().absolute()]
+        contract_source=[cast(CairoFilename, test_file_path.resolve().absolute())]
```

Comment:
I'm not sure about cast required for pyright to work with `NewType`s. There should be a solution that doesn't require casting.

Reply:
It is already resolved. I've changed CairoFilename from NewType to str 😅